### PR TITLE
Create Changelogs “4-8-0-2019-10-21”

### PR DESCRIPTION
**Pull request by alioso**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by alsaihn — APPROVED** [review]

File: src/content/changelog/4-8-0-2019-10-21.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: '4.8.0: 2019-10-21'
```
Comment (yoshizzle) [inline src/content/changelog/4-8-0-2019-10-21.md:2]: This should be 4.7

File: src/content/changelog/4-8-0-2019-10-21.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+date: 2019-11-04T05:00:00.000Z
+version: 4.8.0
```
Comment (yoshizzle) [inline src/content/changelog/4-8-0-2019-10-21.md]: 4.7

**Review by yoshizzle — CHANGES_REQUESTED** [review]

File: src/content/changelog/4-8-0-2019-10-21.md (hunk 1 above)
Comment (nmelehan) [reply src/content/changelog/4-8-0-2019-10-21.md:2]: @yoshizzle the current spec is at 4.7.0, and this update added the object-storage/cancel endpoint. May I ask why the new release would also be 4.7.0?

Comment (yoshizzle) [reply src/content/changelog/4-8-0-2019-10-21.md:2]: oh wait, actually nevermind. I had my repos mixed up.

Nothing to see here, sorry!

**Review by yoshizzle — APPROVED** [review]

File: src/content/changelog/4-8-0-2019-10-21.md (hunk 1 above)
Comment (nmelehan) [reply src/content/changelog/4-8-0-2019-10-21.md:2]: 🙏

**Pull request merged**